### ISSUE TITLE
NeoPixel Sketch Outdated

### ISSUE THREAD

**Issue by armysolo** [FYSETC/FYSETC-BLV-MGN-CUBE#2]
The sketch you've uploaded is old and outdated. It will only work for RRF2. RRF3 has been released for over 18 months and we should be pushing users to use the latest version of software, not something that hasn't been updated in almost 2 years. We've had members from the community update the sketch to be able to work with RRF2 and RRF3 linked below.

Also originally it was required to have a logic level shifter with a Duet 2 Wifi for the NeoPixel rings to function correctly(see New Sketch link). Is this still needed or is there a logic level shifter incorporated to the new board?

[NeoPixel](https://github.com/FYSETC/FYSETC-BLV-MGN-CUBE/tree/main/Firmware/NeoPixel_Ring%20light)
[New Sketch](https://github.com/mule1972/NeoPixelBLVmgn)

**Comment by alienboyxp** [comment]
Hi @armysolo did you have a chance to test the new sketch on Fysetc board? I have the same issue after update to RRF3 and before to start updating if you have any extra information will be welcomed :)

**Comment by armysolo** [comment]
Yes the NeoPixel board works with the New Sketch on RRF3.3 and a Duet 2 WiFi. It does not work with a Duet 3

**Comment by alienboyxp** [comment]
> Yes the NeoPixel board works with the New Sketch on RRF3.3 and a Duet 2 WiFi. It does not work with a Duet 3

Thanks @armysolo !!!
